themes/10up-theme/package.json

```diff
@@ -16,6 +16,9 @@
   },
   "devDependencies": {
     "@10up/stylelint-config": "^1.1.5",
+    "@tailwindcss/aspect-ratio": "^0.4.0",
```

Comment:
@iansvo are we including these by default? It was my understanding that we were going to rather provide documentation around these than assume each project would want to have them implemented?

Reply:
Sorry that was leftover for when I was including them before. Removed in my latest commit.